.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Community Support
+    url: https://github.com/PHPCSStandards/PHP_CodeSniffer/discussions
```

Comment:
Have you considered adding a link to the page that [lists the discussions in the Q&A category](https://github.com/PHPCSStandards/PHP_CodeSniffer/discussions/categories/q-a) or the [page to open a new Q&A discussion](https://github.com/PHPCSStandards/PHP_CodeSniffer/discussions/new?category=q-a) instead of a link to the page that lists all discussions?

I don't have a strong opinion, but I prefer one of the two options I mentioned as they will point the user to something more useful to them to open a new support question. Linking to the page to open a new Q&A discussion is probably what they are looking for in most cases, but I can also see an argument to link to the page that lists the Q&A discussions as users get the chance to see the previous discussions before opening a potentially duplicate one.

Reply:
Fair point.

I'm in two minds about this as there may be discussions in "Show & Tell" and "General" which may be relevant to the support question a well. There may even be closed discussions which are relevant.

Between directly linking to the "open a new discussion" vs "linking to previous discussions", I strongly prefer to link to the list of previous discussions as that has a higher chance of the user being able to solve their issue without even opening a new discussion.

What about updating the link to go to [All discussions except announcements](https://github.com/PHPCSStandards/PHP_CodeSniffer/discussions/categories/announcements?discussions_q=-category%3AAnnouncements+category%3AAnnouncements) ?

Reply:
Hmm.. the downside of that link is that a new discussion will then default to an "Announcement" (counter-intuitive, but looks like that is the GH behaviour)

Reply:
> Between directly linking to the "open a new discussion" vs "linking to previous discussions", I strongly prefer to link to the list of previous discussions as that has a higher chance of the user being able to solve their issue without even opening a new discussion.

Good point!

> What about updating the link to go to [All discussions except announcements](https://github.com/PHPCSStandards/PHP_CodeSniffer/discussions/categories/announcements?discussions_q=-category%3AAnnouncements+category%3AAnnouncements) ?

I believe there is a typo in the link that you shared. It excludes and then tries to include announcements: `-category:Announcements category:Announcements`. The correct link is https://github.com/PHPCSStandards/PHP_CodeSniffer/discussions?discussions_q=-category%3AAnnouncements, and it looks to me to be the best option.

> Hmm.. the downside of that link is that a new discussion will then default to an "Announcement" (counter-intuitive, but looks like that is the GH behaviour)

I don't get this behavior with both links. In both, when I click "New discussion", I see a page to select a discussion category, and "Announcement" is not included in the list. Maybe that is because you have permission to create announcements as an admin of the repo, and I don't?

With your link, I do get the page to select a discussion category with an error at the top that seems to confirm that the difference that we are seeing is related to different permissions that we have:

![Screenshot from 2024-06-04 11-28-38](https://github.com/PHPCSStandards/PHP_CodeSniffer/assets/77215/df4426b1-5d97-42f8-b10c-f49fc1d8011e)

Reply:
Ah! Yes, looks like that is a permission issue. For the link I posted, I only used `-category:Announcements`, GH added the `category:Announcements` itself, I didn't do that and I couldn't get rid of it either (believe me, I tried in so many ways).

Reply:
I've updated the link (again), so hopefully this should be good to go now.

Reply:
It is working as expected for me :raised_hands: